ci/vendor-wit.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+# Script to re-vendor the WIT files that Wasmtime uses as defined by a
+# particular tag in upstream repositories.
+#
+# This script is executed on CI to ensure that everything is up-to-date.
+
+set -ex
+
+dst=crates/wasi/wit/deps
+
+rm -rf $dst
```

Comment:
Maybe check `git diff --exit-code` before this and exit the script before doing any permanent deletions if there are local, uncommitted changes? Just to be safe.

Reply:
Personally I'd prefer to leave that to CI because it makes editing a script like this hard to work with and git is otherwise pretty good about making sure things aren't overwritten.